frontend/src/api/axiosInstance.ts

```diff
@@ -25,7 +26,6 @@ instance.interceptors.response.use(
   (res) => res,
   async (err) => {
     const originalConfig = err.config;
-
     if (err.response.data.message === 'Token expired.') {
       try {
         const response = await axios.get(`${API_URL}auth/refresh/token`, {
```

Comment:
What if bekend changes the message? Is it a good idea to rely on message to check whether the token expired?

Reply:
for now, all errors have the same status code  after that is changed I will update to check by status code